Assets/Scripts/AI/RacerManager.cs

```diff
@@ -6,11 +7,17 @@ public class RacerManager : MonoBehaviour
 {
     private static RacerManager Instance;
 
-    private List<Rigidbody> racerRigidbodies = new List<Rigidbody>();
-    private List<PositionTracker> positionTrackers = new List<PositionTracker>();
+    private static List<Rigidbody> racerRigidbodies = new List<Rigidbody>();
+    private static List<PositionTracker> positionTrackers = new List<PositionTracker>();
```

Comment:
My understanding of static singletons isn't super deep but didn't we deliberately want to use Instance.xyz? Was this causing issues?

Reply:
Jamil and Lin would know more, I just noticed them being added intentionally

Reply:
yes that is how you would make a singleton. This here is not a singleton pattern but that's not to say we need a singleton here. This would actually work although it is modifying what I had planed/used for this class so far